honeybee_grasshopper_radiance/src/HB Annual Daylight.py

```diff
@@ -196,35 +190,25 @@ def ToString(self):
         msg = 'Input _model contains no sensor grids, which will result in a ' \
             'meaningless simulation.\nMake sure that you have assigned grids to ' \
             'the Model with the "HB Assign Grids and Views" component.'
-        give_warning(ghenv.Component, msg)
-        print(msg)
-    all_grids = [g.display_name for g in _model.properties.radiance.sensor_grids]
-    if len(_sensor_grids_) == 0 or _sensor_grids_[0] is None:
-        _sensor_grids_ = all_grids  # use all the Model's sensor grids
-    else:
-        for grid in _sensor_grids_:
-            assert grid in all_grids, \
-                'Sensor grid "{}" was not found in the Model.'.format(grid)
+        raise ValueError(msg)
     assert isinstance(_wea, Wea), 'Expected Wea object. Got {}.'.format(type(_wea))
     try:
         north_vector = to_vector2d(north_)
         north_angle = math.degrees(north_vector.angle_clockwise(Vector2D(0, 1)))
     except AttributeError:  # north angle instead of vector
         north_angle = float(north_)
 
-    # this part is an optional step for each recipe to process the model for luigi input
+    # this part is an optional step for each recipe to process the model
     def default_simulation_path(self):
-        return os.path.join(
-            hb_folders.default_simulation_folder,
-            self._info['inputs']['model'].identifier, 'Radiance')
+        return os.path.join(hb_folders.default_simulation_folder, 'Radiance')
```

Comment:
This seems pretty undesirable here.  So every annual daylight study will overwrite the previous one unless the user specifies a whole folder for the "Run Workflow" component.  This also creates inconsistency with the energy workflows.  can we hold off on making this change until we get the chance to talk and we can implement the same change to both HB-Energy and HB-Radiance.

Reply:
This was not intentional. Feel free to change it to what it is supposed to be.